### PR TITLE
Add a Configuration page to the documentation

### PR DESCRIPTION
**Pull request by Zeitsperre**
This adds a configuration page to the documentation

Related issue #18 

### Changes

* Added page showcasing how to configure your `esgpull` environment
* Reorganized the setup page to show setup commands after installation steps
* Using the `shell` directive and `$` before all commands for docs clarity
* Added a note about a potential bug that can occur when configuring environment

### Discussion

I figured it would be good to add some examples on how to enable downloads and modify configurations to the documentation. There are a few other use cases I would like to have clarified before I can add them to the docs. Will open issues accordingly.

### CODE REVIEW

**Comment by svenrdz** [comment]
Thank you for the contribution @Zeitsperre 
I made a few suggestions, mostly to clarify the login situation, which is only mandatory for a subset of the ESGF data.

Also, for the new configuration page to appear in the deployed documentation, you may want to add a line to the file `docs/mkdocs.yml`. It should be under `nav` at the beginning of the file and mention the filename `configuration.md` with a title for the table of contents.

Thanks again, I appreciate the help and feedback!

**Comment by svenrdz** [comment]
I think my suggested changes are visible now, thanks for pointing it out. I'm not yet familiar with all the fancy features of Github.

File: docs/docs/configuration.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+👍 Config generated at /home/me/.esgpull/config.toml
+```
+
+## Login
+
+In order to download data from ESGF, you will need to provide a valid login and password.
+This can be provided from the command line by running the following:
+
+```shell
+$ egsdpull login
```
[the remark this answers is not in the document]
Comment (Zeitsperre) [reply docs/docs/configuration.md:82]: Woops!

**Comment by Zeitsperre** [comment]
Accidentally double-committed a changed. Reverted via force push.

**Pull request merged**